src/Courses/TopicGrades/WorkbookSelect.tsx

```diff
@@ -78,11 +78,18 @@ export const WorkbookSelect: React.FC<WorkbookSelectProps> = ({grade, onChange,
                 </ListSubheader>
             </Grid>
             <Grid container item xs={12} spacing={2}>
-                {(_.keys(versionMap).length > 1) && // don't show unless multiple versions...
+                {(_.keys(versionMap).length > 0) && // don't show unless multiple versions...
                 <Grid item md={4}>
                     <FormControl className={classes.formControl} fullWidth={true}>
                         <InputLabel id='student-versions'>Viewing Version:</InputLabel>
-                        <Select labelId='student-versions' value={versionKey} onChange={setAttemptsForThisVersion} fullWidth={false} SelectDisplayProps={{style: {display: 'flex', alignItems: 'center', paddingLeft: '10px', paddingRight: '30px'}}}>
+                        <Select 
+                            labelId='student-versions' 
+                            value={versionKey} 
+                            onChange={setAttemptsForThisVersion} 
+                            fullWidth={false} 
+                            SelectDisplayProps={{style: {display: 'flex', alignItems: 'center', paddingLeft: '10px', paddingRight: '30px'}}}
+                            disabled={_.keys(versionMap).length === 1}
```

Comment:
Does this break current if you have 1 submitted version and a current version?

I feel like it should be fine just curious if we tested:
Grading page before submitting
Grading page after submitting
Grading page after closing
Grading page after starting a new version

Reply:
I checked these, it seems to work for current and behave as expected for these cases.